src/views/penalty-details.njk

```diff
@@ -54,7 +54,7 @@
               text: 'Penalty reference'
             },
             hint: {
-              text: 'Enter the full penalty reference. It will be in the format A00000000, PEN1A/00000000 or PEN2A/00000000.'
+              text: 'Enter the full penalty reference. It will be in the format A0000000, PEN1A/00000000 or PEN2A/00000000.'
             },
             errorMessage: validationResult.getErrorForField('userInputPenaltyReference') if validationResult,
             classes: 'govuk-input--penalty-reference',
```

Comment:
Can't put a comment on the actual line, 74 but, that needs to change as well to 7 digits after the A

Reply:
good catch, i've updated that now